src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -62,6 +63,13 @@ class LayerSummary:
         >>> summary.out_size
         [1, 8, 3, 3]
 
+        >>> summary = LayerSummary(model)
+        >>> output = model(input=torch.rand(1, 3, 5, 5))
+        >>> summary.in_size
+        [1, 3, 5, 5]
+        >>> summary.out_size
+        [1, 8, 3, 3]
```

Comment:
Now that you changed the tests, this addition is redundant

Reply:
Fixed